### PR TITLE
faqs enhance

### PR DESCRIPTION
**Pull request by KennethKnows**
faqs enhance

### CODE REVIEW

File: src/components/Faqs.tsx [hunk 1]
```diff
@@ -21,19 +22,42 @@ const FAQsPage: React.FC = () => {
...
+          <div className="max-w-7xl mx-auto px-6 sm:px-8 lg:px-12">
+            <div className="text-start">
+              <h2 className="mt-4 text-3xl text-gray-600 tracking-wider text-primary font-">
+              FAQs
+              </h2>
+            </div>
           </div>
-        ))}
+          {/* Image display with responsive margin and padding */}
+          <div className="mt-8 lg:mt-14 flex justify-center lg:px-14 lg:pl-32">
```
Comment (kynttt) [inline src/components/Faqs.tsx]: too much left padding here. you have lg:px-14 then you also have lg:pl-32. it means you have double padding to your left side. px means padding on x axis, which means left padding has a padding already from this point.

**Review by kynttt — CHANGES_REQUESTED** [review]
check comment

File: src/components/Faqs.tsx (hunk 1 above)
Comment (KennethKnows) [reply src/components/Faqs.tsx]: Resolve issue

**Review by kynttt — CHANGES_REQUESTED** [review]
Check the design in figma, the image box-height must have equal box-height w/ the contents.

**Comment by KennethKnows** [comment]
update, I added the H and W base on the figma.

**Pull request merged**